cc/keystone/auth/plugins/cc_password.py

```diff
@@ -121,18 +119,18 @@ def _authenticate_ews(username, password):
                 raise exception.Unauthorized(msg)
             if response.status_code != 200:
                 LOG.info(
-                    "SAP EWS authentication of '%s' against '%s' was rejected: %s" % (
+                    "SAP Password Verification Service authentication of '%s' against '%s' was rejected: %s" % (
                         username, CONF.cc_password.url, response.reason))
                 msg = _('Invalid username or password')
                 raise exception.Unauthorized(msg)
 
-            LOG.debug("Authenticated %s with EWS." % username)
+            LOG.debug("Authenticated %s with Password Verification Service." % username)
             return True
         except exception.Unauthorized:
             raise
         except Exception as e:
             LOG.error(
-                "SAP EWS authentication of '%s' against '%s' failed: %s" % (
+                "SAP Password Verification Service authentication of '%s' against '%s' failed: %s" % (
                     username, CONF.cc_password.url, e))
             raise exception.Unauthorized('Callout to SAP Exchange for password verification failed.')
```

Comment:
the description of this exception should also be changed

Reply:
any suggestions ?

Reply:
Callout to SAP Password Verification Service failed

Reply:
sry, i dont get. Please update your msg description if the lastest comment is invalid.